packages/fast-components-react-base/src/toolbar-item-group/toolbar-item-group.props.ts

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {
+    ManagedClasses,
+    ToolbarItemGroupClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { Orientation } from "@microsoft/fast-web-utilities";
+
+export interface ToolbarItemGroupManagedClasses
+    extends ManagedClasses<ToolbarItemGroupClassNameContract> {}
+export interface ToolbarItemGroupUnhandledProps
+    extends React.HTMLAttributes<HTMLDivElement> {}
+export interface ToolbarItemGroupHandledProps extends ToolbarItemGroupManagedClasses {
+    /**
+     * The "address" of the group in the toolbar hierarchy.
+     * In toolbars with nested item groups the index can be described with an array
+     * indicating the the index of the child group(s) in succession
+     * ie. [1, 3] places focus on child at index 3 of a group at index 1 of the toolbar root.
+     */
+    itemPath?: number[];
```

Comment:
did you explore other API options for this requirement? This feels rather awkward but I'm not sure what to suggest in place of it.

Reply:
Actually the place for this question is with respect to toolbar props where we have:
initialFocusIndex?: number | number[];

In that case the author would actually need to write the notation.  So maybe in that case we could use id's or a ref?:
initialFocusItemId?: string;

initialFocusItemRef?: React.RefObject<any>

This particular prop should only ever be set programmatically by the parent and not by the author so I've added this to the comment
     * This prop is set programmatically by the parent toolbar/toolbar-item-group and should
     * not be set directly by authors.
What would be better here is a means to set this value in the parent's render fn without suggesting to authors that it's something they should modify.

Reply:
I'm going to go ahead and change the initialFocusIndex to a ref based mechanism as that seems better.

Reply:
Cool sounds good - yea I saw the notation in a few places, the comment was contextual to all. I think refs would feel nicer, but could potentially have trade-offs. Thanks for looking into it